lib/config.sh

```diff
@@ -27,33 +27,33 @@ importEnv () {
 	importOptionsEnv "$_json" || return 1
 	importServicesEnv "$_json" || return 1
 
-	if [[ "$OMNIA_MODE" == "RELAYER" || "$OMNIA_MODE" == "RELAY" ]]; then
+	if [[ "$OMNIA_MODE" == "RELAY" ]]; then
 		importFeeds "$_json" || return 1
 	fi
 }
 
 importMode () {
 	local _json="$1"
-	OMNIA_MODE="$(jq -r '.mode' <<<$_json | tr '[:lower:]' '[:upper:]')"
-	[[ "$OMNIA_MODE" =~ ^(FEED|RELAYER|RELAY){1}$ ]] || { error "Error - Invalid Mode param, valid values are 'FEED' and 'RELAYER'"; return 1; }
+	OMNIA_MODE="$(jq -r '.mode' <<<"$_json" | tr '[:lower:]' '[:upper:]')"
+	[[ "$OMNIA_MODE" =~ ^(FEED|RELAY){1}$ ]] || { error "Error - Invalid Mode param, valid values are 'FEED' and 'RELAY'"; return 1; }
 	export OMNIA_MODE
 }
 
 importSources () {
 	local _json="$1"
-	readarray -t OMNIA_FEED_SOURCES < <(jq -r '.sources[]' <<<$_json)
+	readarray -t OMNIA_FEED_SOURCES < <(jq -r '.sources[]' <<<"$_json")
 	[[ "${#OMNIA_FEED_SOURCES[@]}" -gt 0 ]] || OMNIA_FEED_SOURCES=("gofer" "setzer")
 }
 
 importTransports () {
 	local _json="$1"
-	readarray -t OMNIA_TRANSPORTS < <(jq -r '.transports[]' <<<$_json)
-	[[ "${#OMNIA_TRANSPORTS[@]}" -gt 0 ]] || OMNIA_TRANSPORTS=("transport-spire" "transport-ssb")
```

Comment:
Can we maintain backward compatibility?

Reply:
This is actually a small security improvement,